sycl/include/sycl/accessor.hpp

```diff
@@ -37,7 +37,7 @@
 #include <sycl/property_list.hpp>                     // for property_list
 #include <sycl/range.hpp>                             // for range
 #include <sycl/sampler.hpp>                           // for addressing_mode
-#include <ur_api.h>                                   // for UR_RESULT_ERRO...
+#include <sycl/ur_api.h>                              // for UR_RESULT_ERRO...
```

Comment:
This change makes me think that `ur_api.h` is our own header, but that's not true. I wonder if we need to put it into some `_deps` subfolder for clarity.

Reply:
I've been thinking about this too. I think even `sycl/_deps/ur_api.h` might be questionable, because one might want to use *both* UR and SYCL on the application side and I'd expect user's UR must somehow match the one `libsycl.so` was built with. Maybe I should be deploying these `ur_*.h/.hpp` at root `include/` directory instead.

Does anybody have any ideas here?

Reply:
Implemented the move in the latest revision.

Reply:
Or should it be `ur/ur_*.h` ? @intel/unified-runtime-reviewers

Reply:
Ping @intel/unified-runtime-reviewers to answer the question above. @AlexeySachkov just in case as well.

Reply:
I don't think we ever intend for UR to be used directly by an application so we can eliminate that as a concern. In the UR repo the API header just lives in the root `include` so `ur/ur_api.h` would look a little weird from that perspective

Reply:
I'm not sure if I prefer `_deps/ur_api.h` vs just `ur_api.h`, especially since the latter is the prior spelling of the includes. As such, `_deps/*` move should be in another PR, if at all.